helm/app-operator/templates/network-policy.yaml

```diff
@@ -1,20 +0,0 @@
-kind: NetworkPolicy
```

Comment:
Are you sure you want to remove the network policy?

Reply:
I only renamed the files to be aligned across repos, checking why it marks as deleted